lib/ObjWriter/objwriter.cpp

```diff
@@ -259,7 +259,8 @@ enum CustomSectionAttributes : int32_t {
 };
 
 extern "C" bool CreateCustomSection(ObjectWriter *OW, const char *SectionName,
-                                    CustomSectionAttributes attributes) {
+                                    CustomSectionAttributes attributes,
+                                    const char *ComdatName) {
```

Comment:
ComdateName is for Windows/Coff only.
Can you add an assert at minimum if ComdatName is not null like?:
```
assert(OW->MOFI->getObjectFileType() == OW->MOFI->IsCOFF);
```